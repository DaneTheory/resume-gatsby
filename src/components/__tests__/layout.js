import React from "react"
import renderer from "react-test-renderer"
import Layout from "../layout"
import SocialAccounts from '../socialAccounts'
import IndexPage from '../../pages/index'
describe("Layout", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<IndexPage siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})