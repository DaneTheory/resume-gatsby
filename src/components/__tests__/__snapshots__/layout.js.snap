// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 gwEfcX"
>
  <div
    className="sc-AxjAm iKHSUB"
  >
    <main
      className="StyledBox-sc-13pk1d4-0 gHwNfO sc-AxirZ jDbLra"
    />
  </div>
</div>
`;
